src/components/RNVideoRenderView.js

```diff
@@ -15,12 +15,13 @@ export class RNVideoRenderView extends React.Component {
     // Because "componentDidMount" will be called "immediately after the initial rendering occurs"
     // This is *before* RCTUIManager add this view to register (so that viewForReactTag() can return a view)
     // So we need to dispatch bindVideoView after this function complete
-    setTimeout(() => {
+    this.timeoutId = setTimeout(() => {
```

Comment:
Let's add the issue link here to explain why timeoutId needs to tracked.

Reply:
updated